### DISCUSSION TITLE
Label color with importance

### DISCUSSION THREAD

**Discussion by blaize** [fyne-io/fyne#4030] [Q&A] [answered]
Hello,
Is there any way to specify the importance of a label to make it colored ?
I use canvas.Text with theme.Warning, theme.Success or theme.Error color.
Would be awsome to make a label colored depending of the message importance. Maybe it's already possible and I miss something ?

**Comment by matwachich** [comment, marked as answer]
It's not possible to the standard label.

But you can perfectly emulate this by using a RichText with a single TextSegment, then configure the TextSegment as you wish...

Also, it seems to me like a good idea to be able to set importance to a label...

**Reply by blaize** [reply]
Thanks, I will look at TextSegment. Yep, label with importance could be a feature request.
